src/models/order.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    TigrisCollectionType,
+    TigrisDataTypes,
+    TigrisSchema
+} from "@tigrisdata/core/dist/types";
+
+export interface Order extends TigrisCollectionType {
+    orderId: number;
+    userId: number;
+    productItems: ProductItem[]
+    orderTotal: number;
+}
+
+export interface ProductItem {
```

Comment:
probably just call it `Product`

Reply:
Actually, ignore it - Order is referring to it so it makes sense to call it ProductItem.

Reply:
Product is a different entity. This is a wrapper object part of order.